README.md

```diff
@@ -14,4 +15,4 @@ go read those docs.
 
 If that's unclear or you need more help, try
 [the Bay12 forums thread](http://www.bay12forums.com/smf/index.php?topic=164123)
-or the #dfhack IRC channel on freenode.
+or the #dfhack IRC channel on Libera Chat.
```

Comment:
I might replace all of this with a reference to https://docs.dfhack.org/en/latest/docs/Support.html, which is more up-to-date (and a single place to keep up-to-date)

Reply:
Since it also has the forums on it, I just replaced the paragraph with it (mostly)